zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1554,6 +1597,18 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/ProcessInstanceReferenceItem"
+    CancelProcessInstanceRequest:
+      type: object
+      nullable: true
+      properties:
+        operationReference:
+          description: |
+            A reference key chosen by the user and will be part of all records resulting from this operation.
```

```suggestion
            A reference key chosen by the user that will be part of all records resulting from this operation.
```
🔧 Just a wording suggestion 🙂